README.md

```diff
@@ -11,15 +11,24 @@ If creating a new experiment, see [the docs](docs/new_experiments.md).
 
 ## Experiments
 
-| Name | Status | Results | Run Experiment |
-| --- | --- | --- | --- |
-| [LAMMPS](https://github.com/faasm/experiment-lammps) | :x: MPI Error | [Plots](https://github.com/faasm/experiment-base/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-lammps/blob/master/README.md) |
-| [ParRes Kernels](https://github.com/faasm/experiment-kernels) | :heavy_check_mark: Waiting Plot Spec | [Plots](https://github.com/faasm/experiment-base/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-kernels/blob/master/README.md) |
-| [PyWren](https://github.com/faasm/experiment-pywren) | :clock9: WIP | [Plots](https://github.com/faasm/experiment-master/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-pywren/blob/master/README.md) |
+| Name | Plot Ready | Native Execution | Faasm Execution | Results | Run Experiment |
+| --- | --- | --- | --- | -- | -- |
+| [LAMMPS](https://github.com/faasm/experiment-lammps) | | :white_check_mark: | | [Plots](https://github.com/faasm/experiment-base/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-lammps/blob/master/README.md) |
+| [ParRes Kernels](https://github.com/faasm/experiment-kernels) | :white_check_mark: | :white_check_mark: | :white_check_mark: | [Plots](https://github.com/faasm/experiment-base/blob/master/results/kernels/README.md) | [Instructions](https://github.com/faasm/experiment-kernels/blob/master/README.md) |
+| [PyWren](https://github.com/faasm/experiment-pywren) | | :white_check_mark: | | [Plots](https://github.com/faasm/experiment-master/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-pywren/blob/master/README.md) |
+| [Covid](https://github.com/faasm/experiment-covid) | | :white_check_mark: | | [Plots](https://github.com/faasm/experiment-base/blob/master/plots/README.md) | [Instructions](https://github.com/faasm/experiment-covid/blob/master/README.md) |
 
 ## Quick Start
 
+To run the experiments, first clone this repository and install the
+dependencies:
+```bash
+git clone https://github.com/faasm/experiment-base --recurse-submodules
```

Comment:
Will this go all the way down the submodule tree? We may only need depth of 1 here (to avoid pulling LLVM etc.)

Reply:
I've run into two problems following this approach.
1. `--recurse-submodules --depth 1` recurses _all_ submodules but just the latest commit. I haven't found how to just recurse with recursion depth 1 programatically.
2. Even if I managed to run 1, I also need `faasmtools` (in `faasm/clients/cpp`) for which depth 2 would be required.

I think it's easier to run the `submodule update --init` where necessary.